github-dark.css

```diff
@@ -2924,9 +2927,21 @@
   .bot-identifier, .dashboard-notice .dismiss:hover, .text-gray, a.text-gray,
   td.icon .octicon-file-text, td.icon .octicon-file-symlink-file,
   .UnderlineNav-item, a.UnderlineNav-item, .project-header-link,
-  a.project-header-link, button.project-header-link, td.icon {
+  a.project-header-link, button.project-header-link, td.icon,
+  .js-hide-project-card-details, .js-hide-project-menu {
     color: #949494 !important;
   }
+  /* https://github.com/StylishThemes/GitHub-Dark/issues/640 */
+  .js-hide-project-card-details:hover, .js-hide-project-menu:hover {
+    color: #c0c0c0 !important;
+  }
+  .facebox-close {
+    color: #949494 !important;
+    opacity: .7;
```

Comment:
Please add this selector to the existing `opacity: .75 !important` entry. And also add it to the already defined `color: #949494 !Important;` entry.